schema.json

```diff
@@ -2105,7 +2100,7 @@
       },
       {
         "type": "checkbox",
-        "label": "Show Copyright Info",
+        "label": "Show \"©\", current year and store name",

```

Comment:
Can we use an HTML entity for the copyright symbol
 here to make sure it renders correctly?